files/synchronize.py

```diff
@@ -164,6 +164,11 @@
     default: no
     required: false
     version_added: "2.0"
+  ssh_passwd:
+    description:
+      - Tells rsync to use sshpass with password
+    required: false
+    version_added: "2.0"
```

Comment:
@sivel   as @ritzk said, it was injected by https://github.com/ansible/ansible/pull/11657. 
but not sure  module argument can be used like this, if not, then my PR should be closed, it gets some bad styles.